test/E2E/Helpers.purs

```diff
@@ -0,0 +1,382 @@
+module Test.E2E.Helpers
```

Comment:
Sorry for not communicating it clear, I think we want helpers to be in `src`, but not the functions that are only usable for us, like our passwords.

Reply:
Yeah, sorry for being dumb. ;-) I tried again.